include/UI/UIView.h

```diff
@@ -220,16 +255,45 @@ class UIView
 
   vec2f m_histogram_point_1;
   vec2f m_histogram_point_2;
-  float m_histogram_quantiles[2]; 
+  float m_histogram_quantiles[2];
+
 
-  
   void load_ocean_color_maps();
 
   std::vector<std::string> m_ocean_color_maps_names;
 
   std::string m_color_map_directory;
 
+  float m_animation_speed;
+  std::string m_str_animation_duration;
+  
+  bool m_camera_animation_duration_open;
+  
+  std::vector<float> m_clip_maxs;
+  std::vector<float> m_clip_mins;
+
+
+  bool m_show_clock;
+  float halfScreen;
+  float m_clock_pos_x;
+  float m_clock_pos_y;
+  float m_clock_width;
+  float m_clock_height;
+
+  float m_legend_pos_y;
+
+  std::string m_time_info;
+  std::string m_day_info;
+
+  std::string months[12] = { "January", "February", "March", "April", "May", "June", "July", "August", "September", "October", "November", "December" };
```

Comment:
same question here for this class variable, shouldn't it have the `m_` prefix?

Reply:
I added the m_